<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>page_login</class>
 <widget class="QWidget" name="page_login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>401</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>720</width>
    <height>401</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>720</width>
    <height>401</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>登录</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/logo.png</normaloff>:/logo.png</iconset>
  </property>
  <property name="toolTip">
   <string>这是登录窗口！</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#page_login{
	background-color: rgb(250, 240, 230);
}
#btn_login{
background-color: rgb(135, 206 ,235);
border:2px groove gray;
border-radius:10px;
padding:2px 4px;
border-style: outset;
}
#btn_login:hover{
background-color:rgb(0 ,191 ,255); 
color: black;
}
#btn_login:pressed{
background-color:rgb(135, 206 ,235);
border-style: inset;
}</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>10</y>
     <width>361</width>
     <height>53</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文中宋</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(245, 245, 245);</string>
   </property>
   <property name="text">
    <string>学生科技成果管理信息系统</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_exit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>290</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#btn_exit{
background-color: rgb(135, 206 ,235);
border:2px groove gray;
border-radius:10px;
padding:2px 4px;
border-style: outset;
}
#btn_exit:hover{
background-color:rgb(0 ,191 ,255); 
color: black;
}
#btn_exit:pressed{
background-color:rgb(135, 206 ,235);
border-style: inset;
}

</string>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_login">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>290</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="stu_btn">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>230</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>13</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;楷体&quot;;
background-color: rgb(255, 228, 196);

border: 1px solid gray;/*设置边框的粗细，以及颜色*/
</string>
   </property>
   <property name="text">
    <string>学生</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="manager_btn">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>230</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;楷体&quot;;
background-color: rgb(255, 228, 196);
border: 1px solid gray;/*设置边框的粗细，以及颜色*/
</string>
   </property>
   <property name="text">
    <string>管理员</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>13</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;楷体&quot;;
</string>
   </property>
   <property name="text">
    <string>密  码：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_password">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>170</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(238 ,233 ,233);
border: 1px solid gray;/*设置边框的粗细，以及颜色*/
border-radius: 10px;/*设置圆角的大小*/
padding: 0 8px;/*如果没有内容光标在开始往后移动0.8像素点*/
selection-background-color: darkgray;
   </string>
   </property>
   <property name="maxLength">
    <number>8</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>请输入密码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_user">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(238 ,233 ,233);
border: 1px solid gray;/*设置边框的粗细，以及颜色*/
border-radius: 10px;/*设置圆角的大小*/
padding: 0 8px;/*如果没有内容光标在开始往后移动0.8像素点*/
selection-background-color: darkgray;
   </string>
   </property>
   <property name="maxLength">
    <number>8</number>
   </property>
   <property name="placeholderText">
    <string>请输入用户名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>13</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;楷体&quot;;
</string>
   </property>
   <property name="text">
    <string>用户名：</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>69</y>
     <width>356</width>
     <height>5</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(135, 206, 235);
border: 1px solid gray;/*设置边框的粗细，以及颜色*/</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>0</y>
     <width>5</width>
     <height>401</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(135 ,206, 235);
border: 1px solid gray;/*设置边框的粗细，以及颜色*/</string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>365</x>
     <y>5</y>
     <width>350</width>
     <height>391</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/jiaoshiyi.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>361</width>
     <height>5</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(135, 206, 235);
border: 1px solid gray;/*设置边框的粗细，以及颜色*/</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>396</y>
     <width>361</width>
     <height>5</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(135, 206, 235);
border: 1px solid gray;/*设置边框的粗细，以及颜色*/</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>5</width>
     <height>401</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(135 ,206, 235);
border: 1px solid gray;/*设置边框的粗细，以及颜色*/</string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_6">
   <property name="geometry">
    <rect>
     <x>715</x>
     <y>0</y>
     <width>5</width>
     <height>401</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(135 ,206, 235);
border: 1px solid gray;/*设置边框的粗细，以及颜色*/</string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_7">
   <property name="geometry">
    <rect>
     <x>365</x>
     <y>0</y>
     <width>350</width>
     <height>5</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(135, 206, 235);
border: 1px solid gray;/*设置边框的粗细，以及颜色*/</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_8">
   <property name="geometry">
    <rect>
     <x>365</x>
     <y>396</y>
     <width>350</width>
     <height>5</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(135, 206, 235);
border: 1px solid gray;/*设置边框的粗细，以及颜色*/</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
